pydm/widgets/qtplugins.py

```diff
@@ -241,6 +242,13 @@
                                             icon=ifont.icon("level-up-alt")
                                             )
 
+# Analog Indicator plugin
+PyDMAnalogIndicatorPlugin = qtplugin_factory(PyDMAnalogIndicator,
+                                            group=WidgetCategory.DISPLAY,
+                                            extensions=BASE_EXTENSIONS,
+                                            icon=ifont.icon("level-up-alt")
+                                            )
```

Comment:
Slight indent mismatch
```suggestion
PyDMAnalogIndicatorPlugin = qtplugin_factory(PyDMAnalogIndicator,
                                             group=WidgetCategory.DISPLAY,
                                             extensions=BASE_EXTENSIONS,
                                             icon=ifont.icon("level-up-alt")
                                             )
```